src/blockchainApiConnection/__mocks__/BlockchainQuery.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  Bytes,
+  Option,
+  Tuple,
+  TypeRegistry,
+  U8aFixed,
+  Vec,
+} from '@polkadot/types'
+import AccountId from '@polkadot/types/generic/AccountId'
+import Bool from '@polkadot/types/primitive/Bool'
+import U64 from '@polkadot/types/primitive/U64'
+import U8 from '@polkadot/types/primitive/U8'
+import { Codec } from '@polkadot/types/types'
+import { Constructor } from '@polkadot/util/types'
+
+const TYPE_REGISTRY = new TypeRegistry()
+TYPE_REGISTRY.register({
```

Comment:
Without this, the local tests would not know what `DelegationNodeId` etc. point to.